cpython-unix/patch-apple-cross-3.14.patch

```diff
@@ -0,0 +1,89 @@
+diff --git a/configure.ac b/configure.ac
```

Comment:
This patch may be identical to 3.13, I had to iterate on it. I'll double check and remove it if so.